### PR TITLE
Adding Controller API to explore Zookeeper

### PR DESCRIPTION
**Pull request by kishoreg**
## Description
This PR add a /zookeeper end point to the controller resource. This can be used to view the contents of ZK without having to start zooinspector.
This is a precursor to provide a UI component in the controller to explore ZK contents
Screen Shot 2020-07-11 at 11 35 09 PM

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)
## Release Notes
If you have tagged this as either backward-incompat or release-notes,
you MUST add text here that you would like to see appear in release notes of the
next release.

If you have a series of commits adding or enabling a feature, then
add this section only in final commit that marks the feature completed.
Refer to earlier release notes to see examples of text

## Documentation
If you have introduced a new feature or configuration, please add it to the documentation as well.
See https://docs.pinot.apache.org/developers/developers-and-contributors/update-document

### CODE REVIEW

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotClusterConfigs.java [hunk 1]
```diff
@@ -57,6 +57,17 @@
   @Inject
   PinotHelixResourceManager pinotHelixResourceManager;
 
+  @GET
+  @Path("/cluster/info")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Get cluster Ingo", notes = "Get cluster Info")
```
Comment (mcvsubbu) [inline pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotClusterConfigs.java:63]:
```suggestion
  @ApiOperation(value = "Get cluster Info", notes = "Get cluster Info")
```

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotClusterConfigs.java [hunk 2]
```diff
@@ -57,6 +57,17 @@
...
+  @ApiResponses(value = {@ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 500, message = "Internal server error")})
+  public String getClusterInfo() {
+    ObjectNode ret = JsonUtils.newObjectNode();
+    ret.put("clusterName", pinotHelixResourceManager.getHelixClusterName());
```
Comment (mayankshriv) [inline pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotClusterConfigs.java:67]: Cluster info is just name?

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+    ZNRecord znRecord = pinotHelixResourceManager.readZKData(path);
+    if (znRecord != null) {
+      return new String(_znRecordSerializer.serialize(znRecord), StandardCharsets.UTF_8);
+    }
+    return null;
+  }
+
+  @GET
+  @Path("/zookeeper/ls")
```
Comment (mayankshriv) [inline pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java:79]: Is using `zookeeper` the norm? Wondering if ok to use `zk` for lazy folks like me?

Comment (kishoreg) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java]: Done!

File: pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotClusterConfigs.java (hunk 2 above)
Comment (kishoreg) [reply pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotClusterConfigs.java:67]: yes. We can enhance later. There was no way to get the cluster name

**Pull request merged**